mvpa2/mappers/procrustean.py

```diff
@@ -208,3 +208,10 @@ def _train(self, source):
             d_r = np.linalg.norm(odatas[0] - res_r)/np.linalg.norm(odatas[0])
             debug('MAP_', "%s, residuals are forward: %g,"
                   " reverse: %g" % (repr(self), d_f, d_r))
+
+
+    def _compute_recon(self):
+        """For Procrustean mapper, inverse is transpose.
```

Comment:
There are two tests, one each in test_procrust.py and test_hyperalign.py, that check if the reconstruction is proper. Both would fail if inverse is not computed correctly. Should I write an additional test?